widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/VisitorCodeController.kt

```diff
@@ -1,19 +1,46 @@
 package com.glia.widgets.callvisualizer.controller
 
 import com.glia.widgets.callvisualizer.VisitorCodeContract
+import com.glia.widgets.core.callvisualizer.domain.VisitorCodeRepository
 import com.glia.widgets.core.dialog.DialogController
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
 
-class VisitorCodeController(private val dialogController: DialogController) : VisitorCodeContract.Controller {
+class VisitorCodeController(
+    private val dialogController: DialogController,
+    private val visitorCodeRepository: VisitorCodeRepository
+    ) : VisitorCodeContract.Controller {
 
-    private var view: VisitorCodeContract.View? = null
+    private var disposable: Disposable? = null
+
+    private lateinit var view: VisitorCodeContract.View
 
     override fun setView(view: VisitorCodeContract.View) {
         this.view = view
+        this.view.onSetupComplete()
     }
 
     override fun onCloseButtonClicked() {
         dialogController.dismissCurrentDialog()
     }
 
+    // TODO how to test?
```

Comment:
I think to have this PR ready, we need to handle this TODO.

Reply:
Done